### DISCUSSION TITLE
Possible to block popups?

### DISCUSSION THREAD

**Discussion by TheSoulKeeper** [bromite/bromite#1701] [Q&A]
On some websites, they show small popups at the top/bottom of the page asking whether I would like homescreen shortcut, notifications etc. It's really annoying to keep clicking on "Don't allow" every time. On Firefox+Ublock Origin, I can completely block all popups. Is it possible to do the same with Bromite? I tried using the default adblocking option with Steven Black host, Energized pro host etc, but I couldn't stop those pesky popups. I would really appreciate if the developer can do something to stop those annoying popups for good. Maybe add Chrome extention so we can use Ublock Origin.

**Comment by ghost** [comment]
Blocking these kind of Popups should be possible with User Scripts.
